### PR TITLE
refactor: Move createdTime function into pkg/utils

### PR DESCRIPTION
**Pull request by zeppp**
Signed-off-by: zeppp <zeppp1995@gmail.com>

**1.Describe what this PR did**
Move createdTime function from ps.go to utils.go
**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by zeppp** [comment]
PTAL @allencloud

File: cli/ps.go [hunk 1]
```diff
@@ -4,9 +4,9 @@ import (
 	"fmt"
 	"sort"
 	"strconv"
-	"time"
 
 	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/utils"
 	"github.com/spf13/cobra"
```
Comment (skoo87) [inline cli/ps.go]: add blank line before "cobra"

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -29,3 +43,69 @@ func FormatSize(size int64) string {
 
 	return fmt.Sprintf("%.2f %s", formattedSize, suffixes[count])
 }
+
+// CreatedTime is used to show the time from creation to now.
+func CreatedTime(created string) (s string) {
```
Comment (skoo87) [inline pkg/utils/utils.go]: CreatedTime -> FormatCreatedTime ????  WDYT ?

Comment (allencloud) [inline pkg/utils/utils.go]: What if I input an invalid string as a parameter in this func?

if `asdfghjkl`, what will happen? I think we need to take some exception cases into consideration to improve rubustness.

Comment (zeppp) [reply pkg/utils/utils.go]: Great

Comment (zeppp) [reply pkg/utils/utils.go]: Maybe we should check the validity of input.
But if we use `strconv.FormatInt(time.Now().Unix(), 10)` as input(which must be),the validity will  always be ok.  WDYT? @allencloud

**Comment by 0x04C2** [comment]
Hi @zeppp ， I have issue about this feature, caused by https://github.com/alibaba/pouch/blob/master/apis/server/container_bridge.go#L238

If you run the container, the `Created` status will be wrong. 

```
➜  pouch git:(master) pouch create --name foo1 registry.hub.docker.com/library/busybox ls
container ID: 7c6816759f1c0a120da91bf1a4f492c487f56630a78ba71ce364de6ed1714ced, name: foo1
➜  pouch git:(master) pouch start foo1
➜  pouch git:(master) pouch ps
Name   ID       Status    Image                                            Runtime   Created
foo1   7c6816   stopped   registry.hub.docker.com/library/busybox:latest   runc      about 17519 days ago
``` 

Because the `StartedAt` is date, not timestamp.

```
➜  pouch git:(master) cat /var/lib/pouch/containers/7c6816759f1c0a120da91bf1a4f492c487f56630a78ba71ce364de6ed1714ced/meta.json | jq1
{
  "Args": null,
  "Config": {
    "Cmd": [
      "ls"
    ],
    "Entrypoint": null,
    "Env": null,
    "Image": "registry.hub.docker.com/library/busybox:latest",
    "OnBuild": null,
    "Shell": null,
    "Tty": false
  },
  "HostConfig": {
    "Runtime": "runc"
  },
  "Id": "7c6816759f1c0a120da91bf1a4f492c487f56630a78ba71ce364de6ed1714ced",
  "Mounts": null,
  "Name": "foo1",
  "State": {
    "FinishedAt": "2017-12-19 10:51:30.681763441 +0800 CST m=+566.484707937",
    "Pid": -1,
    "StartedAt": "2017-12-19 10:51:30.56459485 +0800 CST m=+566.367539349",
    "Status": "stopped"
  }
}
```

**Comment by zeppp** [comment]
@0x04C2  Thanks, i will fix it soon. 
Should I change all into timestamp instead date? @allencloud

File: pkg/utils/utils.go (hunk 2 above)
Comment (allencloud) [reply pkg/utils/utils.go]: We do need the check. We need to handle exceptions.

Comment (zeppp) [reply pkg/utils/utils.go]: Ok,i will refactor the function and fix the bug.

**Comment by allencloud** [comment]
CI fails, please go fmt your code:
```
daemon/mgr/container.go
pkg/utils/utils_test.go
please format Go code with 'gofmt -s -w'
Makefile:38: recipe for target 'fmt' failed
make: *** [fmt] Error 1
```

**Pull request closed without merging**